web/frontend/src/language/fr.json

```diff
@@ -151,71 +151,71 @@
     "errorAddUser": "Erreur lors de l'ajout de l'utilisateur",
     "successRemoveUser": "Utilisateur supprimé avec succès!",
     "errorRemoveUser": "Erreur lors de la suppression de l'utilisateur",
     "errorFetchingUsers": "Erreur lors de la récupération des utilisateurs",
-    "voteFailure": "Votre vote n'a pas été pris en compte. C'est possible que le sondage a été fermé ou annulé. Essayez de rafraichir la page.",
-    "ballotFailure": "Une erreur est survenu lors de l'envoi de votre ballot. Merci de contacter l'administrateur de ce site. ",
-    "incompleteBallot": "Certaines réponses ne sont pas complète.",
+    "voteFailure": "Votre vote n'a pas été pris en compte. Il est possible que le sondage ait été fermé ou annulé. Essayez de rafraîchir la page.",
+    "ballotFailure": "Une erreur est survenue lors de l'envoi de votre ballot. Merci de contacter l'administrateur du site. ",
+    "incompleteBallot": "Certaines réponses ne sont pas complètes.",
     "selectMin": "Selectionnez {{minSelect}} {{singularPlural}}. ",
     "selectMax": "Selectionnez au moins {{maxSelect}} {{singularPlural}}. ",
     "selectBetween": "Selectionnez entre {{minSelect}} et {{maxSelect}} réponses. ",
-    "minSelectError": "Vous devez sélectionné au moins {{min}} {{singularPlural}}. ",
-    "maxSelectError": "Vous pouvez pas sélectionné plus de {{max}} réponses. ",
+    "minSelectError": "Vous devez sélectionner au moins {{min}} {{singularPlural}}. ",
+    "maxSelectError": "Vous ne pouvez pas sélectionner plus de {{max}} réponses. ",
     "fillText": "Remplissez {{minText}} {{singularPlural}}. ",
     "minText": "Remplissez au moins {{minText}} {{singularPlural}}. ",
     "minTextError": "Vous devez remplir au moins {{minText}} {{singularPlural}}. ",
-    "maxTextChars": "Les réponses doivent être au maximum long de {{maxLength}} charactères. ",
+    "maxTextChars": "Les réponses doivent avoir au maximum {{maxLength}} caractères. ",
     "regexpCheck": "Les réponses doivent être sous la forme: {{regexp}}. ",
     "singularAnswer": "réponse",
     "pluralAnswers": "réponses",
     "rankRange": "La réponse doit être entre 1 et {{max}}. ",
     "castVote": "Voter",
     "voteExplanation": "Vous pouvez voter autant de fois que vous le souhaitez tant que le sondage est ouvert. Seul votre dernier vote sera pris en compte.",
-    "noVote": "Il y a actuellement rien à voter.",
-    "voteAllowed": "Vous êtes autorisés à voter sur les sondages ci-dessous. Cliquez sur le titre d'un sondage pour afficher son bulletin de vote et voter.",
-    "displayResults": "Les résultats de(s) sondage(s) listé(s) ci-dessous sont disponibles. Clique sur le titre d'un sondage pour y avoir accès.",
-    "noResultsAvailable": "Il y a actuellement aucuns résultats disponibles.",
+    "noVote": "Il n'y a actuellement rien à voter.",
+    "voteAllowed": "Vous êtes autorisé à voter sur les sondages ci-dessous. Cliquez sur le titre d'un sondage pour afficher son bulletin de vote et voter.",
```

Comment:
```suggestion
    "voteAllowed": "Vous êtes autorisés à voter sur les sondages ci-dessous. Cliquez sur le titre d'un sondage pour afficher son bulletin de vote et voter.",
```

Reply:
Idem plus haut.

Reply:
Je ne comprends pas ton commentaire. Ici c'est le verbe "être" tout seul, alors ça s'accorde avec le sujet qui est en pluriel. Donc il y a un "s". Ou est-ce que j'ai loupé quelque chose?

PS: j'aime bien pour une fois discuter sur la grammaire - normalement c'est: "I would put a map here" - "but that makes it more complicated" - "no, it's more KISS and should be done" - "on the contrary, it makes it more complicated" - aso :)